azure/HandsOnTutorial/credentials.fsx

```diff
@@ -1,7 +1,7 @@
-﻿#I "../../packages/MBrace.Azure.Standalone/tools"
```

Comment:
I've updated this now, could you check it?  You're correct that it was loading the things from a different directory, interesting that that happens, I hadn't realized that.